.moai/specs/SPEC-PIPELINE-FANOUT-ACTIVATION-001/acceptance.md

```
@@ -0,0 +1,492 @@
+ .claude/skills/moai/workflows/plan/spec-assembly.md \
+ .claude/skills/moai/workflows/run/phase-execution.md \
+ .claude/skills/moai/workflows/run/task-decomposition.md \
+ .claude/skills/moai/workflows/sync.md \
+ .claude/skills/moai/workflows/sync/quality-gates-quality.md \
+ .claude/skills/moai/workflows/sync/doc-execution.md ; do
+  echo "local=$(grep -cE "$RE" "$f") tmpl=$(grep -cE "$RE" "internal/template/templates/$f") $f"
+done
+```
+
+- Observed baseline:
+  ```
```

markdownlint-cli2 (0.23.1)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)